pom.xml

```diff
@@ -313,9 +325,9 @@
                 <requireUpperBoundDeps>
                   <excludes combine.children="append">
                     <exclude>commons-lang:commons-lang</exclude>
+                    <exclude>org.apache.commons:commons-collections4</exclude>
                     <exclude>org.codehaus.plexus:plexus-archiver</exclude>
                     <exclude>org.codehaus.plexus:plexus-container-default</exclude>
-                    <exclude>org.ow2.asm:asm</exclude>
```

Comment:
Miscellaneous cleanup: realized that this was unnecessary while updating this list.